packages/components/tests/dummy/app/templates/components/form/text-input.hbs

```diff
@@ -215,6 +232,47 @@
     </SG.Item>
   </Shw::Grid>
 
+  <Shw::Text::H3>Password</Shw::Text::H3>
```

Comment:
Now that I've added the `Types` above, do you think this section becomes redundant?

Reply:
Indeed, they feel redundant now so I removed the 'Password' section in the showcase